src/jetstream/default.config.properties

```diff
@@ -29,4 +29,10 @@ SMTP_PORT=1025
 SMTP_USER=
 TEMPLATE_DIR=./templates
 INVITE_USER_CLIENT_ID=
-INVITE_USER_CLIENT_SECRET=
\ No newline at end of file
+INVITE_USER_CLIENT_SECRET=
```

Comment:
This file should be excluded from merge to disable this functionality until front-end changes are complete.

Reply:
After other changes have been implemented and gates passed, will remove default config file from this PR entirely as this feature currently has no corresponding front end changes. Will update default config in the pending frontend change - feature will then be complete and config options fully usable.